<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LkEngineEditor</class>
 <widget class="QMainWindow" name="LkEngineEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LK-Engine 版本0.001</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openSolution"/>
   <addaction name="separator"/>
   <addaction name="saveSolution"/>
   <addaction name="separator"/>
   <addaction name="openSceneCfg"/>
   <addaction name="separator"/>
   <addaction name="closeSceneCfg"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="ops"/>
    <addaction name="cls"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>场景配置</string>
    </property>
    <addaction name="actionfsasdf"/>
    <addaction name="actiondasdasd"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menu"/>
  </widget>
  <action name="openSolution">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>打开解决方案</string>
   </property>
  </action>
  <action name="saveSolution">
   <property name="text">
    <string>保存解决方案</string>
   </property>
  </action>
  <action name="openSceneCfg">
   <property name="text">
    <string>打开基础场景配置</string>
   </property>
  </action>
  <action name="closeSceneCfg">
   <property name="text">
    <string>关闭基础场景配置</string>
   </property>
  </action>
  <action name="ops">
   <property name="text">
    <string>打开解决方案</string>
   </property>
  </action>
  <action name="cls">
   <property name="text">
    <string>保存解决方案</string>
   </property>
  </action>
  <action name="actionfsasdf">
   <property name="text">
    <string>打开场景配置</string>
   </property>
  </action>
  <action name="actiondasdasd">
   <property name="text">
    <string>关闭场景配置</string>
   </property>
  </action>
  <action name="actiondasdas">
   <property name="text">
    <string>dasdas</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
